bin/dashboard/index.js

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+const { Command, UsageError } = require(`clipanion`);
+const fs = require('fs-extra');
+const path = require('path');
+
+const JSON_PATH = path.join(__dirname, '../../assets/dashboard.json');
+
+class Dashboard extends Command {
+    async execute() {
+        if (!this.prefix) {
+            throw new UsageError('Missing --prefix option.');
```

Comment:
If we are requiring the prefix for the generated dashboard inside the datadog , I wonder if we should make it a required parameter. Currently this might not work if someone uses the default prefix of `""` since their metric names would begin with a `.`

https://github.com/DataDog/build-plugin/blob/5d437ddef97b3382605f156d0af23c92bab988c3/src/hooks/datadog/README.md#datadogprefix

Reply:
Good point.
I'll make it accept no prefix.